runhouse/resources/hardware/launcher_utils.py

```diff
@@ -156,7 +208,12 @@ def up(cls, cluster, verbose: bool = True, force: bool = False):
         }
 
         if verbose:
-            data = cls.run_verbose(base_url=cls.LAUNCH_URL, payload=payload)
+            data = cls.run_verbose(
+                log_processor=cls.log_processor,
```

Comment:
Does this need to be set? Or can it be referenced as `self.log_processor` within the `run_verbose` method?

Reply:
ah true there aren't actually any properties that differ betwen launch / teardown